src/main/java/io/kabanero/api/InstanceEndpoints.java

```diff
@@ -124,4 +127,37 @@ public Response isAdmin(@PathParam("instanceName") String instanceName) throws I
         return Response.ok(body).build();
     }
 
-}
\ No newline at end of file
+    @GET
+    @Path("{instanceName}/team/{wantedTeamName}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response isAdmin(@PathParam("instanceName") String instanceName, @PathParam("teamName") String wantedTeamName) throws IOException, ApiException, GeneralSecurityException {
+        UserProfile userProfile = UserProfileManager.getUserProfile();
+        String token = userProfile.getAccessToken();
+        GitHubClient client = new GitHubClient();
+        client.setOAuth2Token(token);
+
+        Kabanero instance = KabaneroClient.getAnInstance(instanceName);
+        if (instance == null) {
+            return Response.status(404).entity(new ResponseMessage(instanceName + " not found")).build();
+        }
+
+        JsonObject body = new JsonObject();
+        String instanceGithubOrg = instance.getSpec().getGithub().getOrganization();
+        TeamService teamService = new TeamService(client);
+
+        for (Team orgTeam : teamService.getTeams(instanceGithubOrg)) {
+            if (wantedTeamName.equals(orgTeam.getName())) {
+                List<User> kabaneroTeamMembers = teamService.getMembers(orgTeam.getId());
+                JsonArray jsonArray = new Gson().toJsonTree(kabaneroTeamMembers).getAsJsonArray();
+                body.add("members", jsonArray);
+                if (kabaneroTeamMembers.size() == 0) {
+                    return Response.status(404)
+                            .entity(new ResponseMessage(wantedTeamName + " currently does not have any members")).build();
+                }
+            } else {
+                return Response.status(404).entity(new ResponseMessage(wantedTeamName + " team not found")).build();
```

Comment:
if the first orgTeam isn't the one we're looking for this will pre-maturely return a 404 instead of looping to the next orgTeam and checking against that one.

I would just have the last line of the function return the 404 response. And in the loop return the actual data if `wantedTeamName.equals(orgTeam.getName())`